WordPress/WordPressTest/PostCardCellTests.swift

```diff
@@ -82,7 +82,7 @@ class PostCardCellTests: XCTestCase {
 
         postCell.configure(with: post)
 
-        XCTAssertEqual(postCell.dateLabel.text, "just now")
+        XCTAssert(postCell.dateLabel.text?.isEmpty == false)
```

Comment:
Mixed feelings about this. I like how you made it language independent. At the same time, checking that the string is not empty may pass in any number of scenarios, including someone accidentally hardcoding a value in the text. 🤔 

Another option could be to check that the result is the same running the input `Date` through the formatter method. That, on the other hand, would bind this code to the implementation detail that we're using the formatter 😫

This is a tricky problem... 🤔 

I think maybe updating to "now" and bank on the fact they'll always run in English might make it more robust. "_bank on the fact they'll always run in English_" is actually something we could enforce in the Test Plan that runs these tests 🤔 

Also, nitpick:

```suggestion
        XCTAssertFalse(postCell.dateLabel.text?.isEmpty)
```

Same thing, but with one less operation. The result if the test fails is, unfortunately, equally uninformative, but that's an XCTest limitation not an issue with the test code 😞 

<img width="844" alt="Screen Shot 2021-06-16 at 9 52 01 am" src="https://user-images.githubusercontent.com/1218433/122137704-9d9b3200-ce88-11eb-9c69-405aac6f65bb.png">